doc/source/guide/plotting.rst

```diff
@@ -169,47 +169,89 @@ highlight a region of interest, and change the plot title.
     plt.colorbar()
     plt.show()
 
+
+Plotting Maps with wcsaxes
+--------------------------
+
 By default :ref:`map` uses the `wcsaxes <http://wcsaxes.readthedocs.org/>`_
 package to improve the representation of world coordinates. In the
-examples above the axes were normal matplotlib axes.
+examples above the axes were normal matplotlib axes, because they were explictily
+created.
+If wcsaxes is installed, then calling `~sunpy.map.GenericMap.plot` or 
+`~sunpy.map.GenericMap.peek()` will default to using wcsaxes for plotting.
 To create a custom `wcsaxes.WCSAxes` instance do the following ::
```

Comment:
A slight re-organization of the text above is suggested:

In the map plotting examples above the axes were normal matplotlib axes, because they were explicitly
created. (@cadair - to what does the "they" refer to? - the normal matplotlib axes?)

By default :ref:`map` checks if the `wcsaxes <http://wcsaxes.readthedocs.org/>`_ 
package has been installed.  If it is installed, then wcsaxes is used to improve the representation of world coordinates, and calling `~sunpy.map.GenericMap.plot` or 
`~sunpy.map.GenericMap.peek()` will use wcsaxes for plotting.